inception-recommendation-api/src/main/java/de/tudarmstadt/ukp/inception/recommendation/api/model/Recommender.java

```diff
@@ -180,46 +186,35 @@ public void setTraits(String aTraits)
     {
         traits = aTraits;
     }
-    
+
     @Override
-    public int hashCode()
-    {
-        final int prime = 31;
-        int result = 1;
-        result = prime * result + ((id == null) ? 0 : id.hashCode());
-        result = prime * result + ((name == null) ? 0 : name.hashCode());
-        return result;
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Recommender that = (Recommender) o;
+        return Objects.equals(id, that.id) &&
```

Comment:
IMHO hashcode and equals should use name/project ("business" primary key) - id should not be included.

Reply:
I just used what was already there. If you want it to change then I will change it.

Reply:
If you prefer, a separate issue/PR can be assigned, but I think it should be addressed.

Reply:
I created an issue for that.